docs/supported_apis/series_supported.rst

```diff
@@ -236,35 +245,39 @@ the related section on :doc:`Defaulting to pandas </supported_apis/index>`.
-| ``max``                     | Y                               |                                                    |
+| ``max``                     | Y                               | **Omnisci**:``P``, only default params supported,  |
+|                             |                                 | otherwise ``D``                                    |
 +-----------------------------+---------------------------------+----------------------------------------------------+
 | ``mean``                    | P                               | Modin defaults to pandas if given the ``level``    |
 |                             |                                 | param.                                             |
+|                             |                                 | **Omnisci**:``P``, ``D`` for ``level``, ``axis``,  |
+|                             |                                 | 'skipna' and 'numeric_only' params                 |
 +-----------------------------+---------------------------------+----------------------------------------------------+
 | ``median``                  | P                               | Modin defaults to pandas if given the ``level``    |
 |                             |                                 | param.                                             |
 +-----------------------------+---------------------------------+----------------------------------------------------+
 | ``memory_usage``            | Y                               |                                                    |
 +-----------------------------+---------------------------------+----------------------------------------------------+
-| ``min``                     | Y                               |                                                    |
+| ``min``                     | Y                               | **Omnisci**:``P``, only default params supported,  |
+|                             |                                 | otherwise ``D``                                    |
 +-----------------------------+---------------------------------+----------------------------------------------------+
-| ``mod``                     | Y                               |                                                    |
+| ``mod``                     | Y                               | **Omnisci**:``Y``                                  |
 +-----------------------------+---------------------------------+----------------------------------------------------+
 | ``mode``                    | Y                               |                                                    |
 +-----------------------------+---------------------------------+----------------------------------------------------+
-| ``mul``                     | Y                               |                                                    |
+| ``mul``                     | Y                               | **Omnisci**:``Y``                                  |

```

Comment:
please sync this with DataFrame supported operations. Notes for Series' binary ops seems to be correct, but they are unsynced with the DataFrame ones (e.g. mul is stated as "Y" here, but in DF notes it's unmarked, meaning that it's "D")

Reply:
Yeah, it says "See ``add``" where Omnisci is mentioned :) I'll add more clarity

Reply:
Done, hopefully